test_runner/fixtures/zenith_fixtures.py

```diff
@@ -358,7 +418,8 @@ def zenith_binpath(base_dir):
 
 @zenfixture
 def pg_distrib_dir(base_dir):
-    """ find the postgress install """
+    """ Find the postgress install. """
```

Comment:
typo

Reply:
Done. I guess we could run codespell on CI.